web_app/dist/main.css

```diff
@@ -0,0 +1,117 @@
+html {
+    background: #93ca68;
+    background-image: url(images/bgLogo.png);
+    background-repeat: repeat;
+}
+
+body {
+    background: #fff;
+    color: #545454;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 18px;
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 70%;
+    padding: 2em 2em 4em;
+    border-radius: 10px;
+    box-shadow: 5px 25px 20px -10px #656565; 
+}
+    
+
+h3 {
+    color: whitesmoke;
+}
+
+/* Change the color of links on hover */
```

Comment:
I'd say comment is useless, behavior of this is obvious by looking at the code (here and elsewhere)